common/flatpak-dir.c

```diff
@@ -9677,7 +9801,19 @@ flatpak_dir_get_locale_subpaths (FlatpakDir *self)
       if (flatpak_dir_is_user (self))
         subpaths = flatpak_get_current_locale_subpaths ();
       else
-        subpaths = g_new0 (char *, 1);
+        {
+          GDBusProxy *proxy = get_accounts_dbus_proxy ();
+          if (proxy)
+            {
+              subpaths = get_locale_subpaths_from_accounts_dbus (proxy);
```

Comment:
This will do N+1 (N==number of user accounts) synchronous dbus calls on the main loop every time this is called, which is several times during an install. At the very least there needs to be some level of caching here.

But there is also a general question about scalability of this. On a single-user laptop this seems ok, but what happens on e.g. a company laptop with ldap configured, etc. How many users could there be on such a machine?